dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/AbstractTask.java

```diff
@@ -106,7 +109,7 @@ protected abstract T doRun(TaskRunContext context, @Nonnull ByteSink sink, @Nonn
 
   @Override
   public T run(TaskRunContext context) throws Exception {
-    if (!createTarget) {
+    if (TargetInitialization.DO_NOT_CREATE == targetInitialization) {
```

Comment:
nit: would be more naturally to see
`if (targetInitialization == TargetInitialization.DO_NOT_CREATE)`